### PR TITLE
Stargaze when gazed on black point

### PR DESCRIPTION
**Pull request by tjiyer**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1655)

### CODE REVIEW

File: english/experience/black-point/black-point.md [hunk 1]
```diff
@@ -4,4 +4,6 @@ However, nothing happen, it's just a little black point.
 
 What do you do now?
 
-[Open your eyes](../../marshmallow.md)
\ No newline at end of file
+[Open your eyes](../../marshmallow.md)
+
+[Concentrate harder on the black point] (/stargaze.md)
```
Comment (casey-collab) [inline english/experience/black-point/black-point.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/experience/black-point/stargaze.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+When you concentrate on the black point for some more time you see the point transform.
```
Comment (casey-collab) [inline english/experience/black-point/stargaze.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/experience/black-point/stargaze.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by tjiyer** [comment]
Made changes please review @casey-collab

**Comment by casey-collab** [comment]
Looks great! :night_with_stars:

**Pull request merged**